src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -646,7 +646,8 @@ def make_protocols(self, entry):
         for protocol in entry["contents"]["protocols"]:
             translated_process = {
                 "libraryConstructionApproach": protocol.get("library_construction_approach", []),
-                "instrumentManufacturerModel": protocol.get("instrument_manufacturer_model", [])
+                "instrumentManufacturerModel": protocol.get("instrument_manufacturer_model", []),
+                "pairedEnd": protocol.get("paired_end", None),
```

Comment:
Why is the default different from the above two properties?

Reply:
Ah, I think i was looking at make_specimen() as a guide which uses None as the default for all the properties.